postprocessor/src/main/java/com/azure/autorest/postprocessor/Postprocessor.java

```diff
@@ -118,7 +99,7 @@ public boolean processInternal() {
             } else if (className.startsWith("src") && className.endsWith(".java")) {
                 customizationClass = loadCustomizationClassFromJavaCode(className, getBaseDirectory(), logger);
             } else {
-                customizationClass = loadCustomizationClassFromReadme(className, readme);
+                throw new RuntimeException("Unable to load customization class.");
```

Comment:
why this change about readme?

Reply:
This is removing the ability to configure code customizations using a Java code block in the README. This is pushing the requirement to use a proper Java source code file or precompiled JAR for code customizations.

Reply:
Got it. This feature is likely rarely used.